Lux/src/image_loader.cpp

```diff
@@ -0,0 +1,26 @@
+// image_loader wraps low level image I/O routines. Implementation can be changed without affecting image classes.
+// Using stb_image, C-style pointer is created when class is initialized with a file name, and freed in the destructor.
+// Copy constructor and operator= are disabled to prevent segmentation faults or memory leaks
+
+#define STB_IMAGE_IMPLEMENTATION
+#include "../../stb_image/stb_image.h"
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include "../../stb_image/stb_image_write.h"
+#include "image_loader.hpp"
+
+image_loader :: image_loader( const std::string& filename ) {
+    stbi_ptr = stbi_load( filename.c_str(), &xsiz, &ysiz, &channels, 0 );
```

Comment:
can move this into colon area i think

Reply:
Made this change - compiles and runs properly. Will incorporate.